pizza_cutter/des_pizza_cutter/_coadd_slices.py

```diff
@@ -555,3 +567,132 @@ def _coadd_slice_inputs(
         interp_se_frac,
         resampled_datas,
     )
+
+
+@functools.lru_cache(maxsize=1)
+def _get_gaia_stars(*, fname):
+    """
+    load the gaia stars
+
+    Parameters
+    -----------
+    fname: str
+        path to the gaia star catalog
+    """
```

Comment:
```suggestion
```

Reply:
It looks like you are suggesting I remove documentation

Reply:
for this small private function, yes. the variable names are good enough that it is self-documenting and the docstring is distracting here. :)